### DISCUSSION TITLE
Unable to read a pdf file

### DISCUSSION THREAD

**Discussion by PriyankaJain-1998** [jsvine/pdfplumber#535] [Ask for help with specific PDFs]
Unable to read a pdf file. 
![image](https://user-images.githubusercontent.com/58905207/140695033-a3d7e631-5924-4281-8377-0b87d8fc89d6.png)
After searching, opening a pdf with particular format like 'r','wb' etc are suggested but still issue persist. 
please help.

**Comment by jsvine** [comment]
Hi @PriyankaJain-1998, and thanks for your interest in this library. Unfortunately, it's not possible to troubleshoot your situation without more context. Could you provide the code that's raising this TypeError?
